<!--**************************************-->
<!--* Generated from MathBook XML source *-->
<!--*    on 2016-04-29T09:40:02-07:00    *-->
<!--*                                    *-->
<!--*   http://mathbook.pugetsound.edu   *-->
<!--*                                    *-->
<!--**************************************-->
<article class="paragraph"><h5 xmlns:b64="https://github.com/ilyakharlamov/xslt_base64" class="heading"><span class="type">Paragraph</span></h5>
In Preview <a href="S_1.7.LimContDiff.html#PA_1.7" alt="Paragraphs  Preview Activity" title="Paragraphs  Preview Activity">Activity </a>, the function \(f\) given in <a knowl="./knowl/F_1.7.PA1.html" knowl-id="xref-F_1.7.PA1" alt="Figure 1.7.1 " title="Figure 1.7.1 ">Figure 1.7.1</a> only fails to have a limit at two values: at \(a = -2\) (where the left- and right-hand limits are 2 and \(-1\), respectively) and at \(x = 2\), where \(\lim_{x \to 2^+} f(x)\) does not exist). Note well that even at values like \(a = -1\) and \(a = 0\) where there are holes in the graph, the limit still exists.
</article><span class="incontext"><a href="S_1.7.LimContDiff.html#p-242">in-context</a></span>
